packages/fxa-auth-server/lib/senders/emails/styles.ts

```diff
@@ -27,7 +27,7 @@ span {
   line-height: 18px !important;
 }
 .app-badges img {
-    width: 152px !important;
+  width: 152px !important;
```

Comment:
While this fix looks correct, you can probably take out this file from the commit (same `git checkout` command as before) since you'll still conflict with yourself in the CSS PR.